src/launch.c

```diff
@@ -307,7 +307,8 @@ int client_launch_exec (int forkit, char *dir, char* argv[], struct server *s) {
 					CLIENT_ERROR_MSG_HEAD, argv[0], g_strerror (errno));
 
 			error_out:
-			write (pipefds[1], msg, strlen (msg) + 1);
+			// HACK: Silence: ignoring return value of ‘write’ declared with attribute ‘warn_unused_result’.
+			if (write (pipefds[1], msg, strlen (msg) + 1)) {}
 			close (pipefds[1]);
```

Comment:
@zturtleman do you think we can or should skip the `close()` if `write()` fails here? 

Here we are already in an error handling code, so there is nothing to really do if such `write()` errors out as well.

But I wonder if we should take care of this `close()` call.

I guess we can and still should close a pipe even if we cannot write to it.

Reply:
I believe calling close() is safe/correct even if performing an action failed on the file descriptor.

Reply:
Thanks for the answer.